test/wallet-test.js

```diff
@@ -70,10 +74,26 @@ function fakeBlock(height) {
     time: 500000000 + (height * (10 * 60)),
     bits: 0,
     nonce: 0,
-    height: height
+    height: height,
+    version: 0,
+    witnessRoot: Buffer.alloc(32),
+    treeRoot: Buffer.alloc(32),
+    reservedRoot: Buffer.alloc(32),
+    extraNonce: Buffer.alloc(24),
+    mask: Buffer.alloc(32)
```

Comment:
I assume we need all these values to make `walletDB.removeBlock()` work?

Reply:
These values are required to create the `ChainEntry` objects needed to pass to `wdb.addBlock`. See the helper functions `curEntry` and `nextEntry` that correspond to `curBlock` and `nextBlock`

Reply:
👍